docs/iot/sample/iot-sample.md

```diff
@@ -0,0 +1,538 @@
+        resp_rep->set_bool_value(resp_rep, KEY_MUTE, g_mute);
+    }
+
+    if (req_msg->has_property_key(req_msg, KEY_VOLUME_COMMAND))
+        WARN("key[%s] exists..Ignore It!", KEY_VOLUME_COMMAND);
+
+    return true;
+}
+
+/* Handle: for setting request on volume */
+bool
+handle_set_request_on_volume(st_things_set_request_message_s *req_msg, st_things_representation_s *resp_rep)
+{
+    DBG("current g_volume: [%lld], g_mute:[%d]", g_volume, g_mute);
+
+    int64_t ivalue;
+    if (req_msg->rep->get_int_value(req_msg->rep, KEY_VOLUME, &ivalue)) {
+        DBG("requested volume: [%lld]", ivalue);
+        g_volume = ivalue;
+        float temp_vol = (float) (g_volume/100.);
+        user_player_set_volume(temp_vol);
+        resp_rep->set_int_value(resp_rep, KEY_VOLUME, g_volume);
+    }
+
+    bool bvalue;
+    if (req_msg->rep->get_bool_value(req_msg->rep, KEY_MUTE, &bvalue)) {
+        DBG("requested mute: [%d]", bvalue);
+        g_mute = bvalue;
+        user_player_set_mute(g_mute);
+        resp_rep->set_bool_value(resp_rep, KEY_MUTE, g_mute);
+    }
+
+    char *str_value = NULL;
+    if (req_msg->rep->get_str_value(req_msg->rep, KEY_VOLUME_COMMAND, &str_value)) {
+        DBG("requested volume_cmd: [%s]", str_value);
+
+        strncpy(g_volume_cmd, str_value, VALUE_STR_LEN_MAX);
+        if (0 == strncmp(g_volume_cmd, VALUE_VOLUME_COMMAND_INC, strlen(VALUE_VOLUME_COMMAND_INC)))
+            DBG("TODO");
+        else if (0 == strncmp(g_volume_cmd, VALUE_VOLUME_COMMAND_DEC, strlen(VALUE_VOLUME_COMMAND_DEC)))
+            DBG("TODO");
+        else if (0 == strncmp(g_volume_cmd, VALUE_VOLUME_COMMAND_MAX, strlen(VALUE_VOLUME_COMMAND_MAX)))
+            DBG("TODO");
+        else if (0 == strncmp(g_volume_cmd, VALUE_VOLUME_COMMAND_MIN, strlen(VALUE_VOLUME_COMMAND_MIN)))
+            DBG("TODO");
+        else
+            ERR("Not supported command");
+
+        resp_rep->set_str_value(resp_rep, KEY_VOLUME_COMMAND, g_volume_cmd);
+        free(str_value);
+    }
+
+    st_things_notify_observers(req_msg->resource_uri);
+
+    return true;
+}
+
+/* Handle: reset request*/
+bool
+handle_reset_request()
+{
+    bool confirmed = true;
+    DBG("confirmed : [%d]", confirmed);
+
+    return confirmed;
+}
+
+/* Handle: reset result */
+void
+handle_reset_result(bool result)
+{
+    DBG("result : [%d]", result);
+}
+
+/* Handle: ownership transfer request */
+bool
+handle_ownership_transfer_request()
+{
+    bool confirmed = true;
+    DBG("confirmed : [%d]", confirmed);
+
+    return confirmed;
+}
+
+/* Handle: for things status change */
+void
+handle_things_status_change(st_things_status_e things_status)
+{
+    DBG("things_status : [%d]", things_status);
+}
+```
+
+## Test
+
+To test the sample:
+
+1.  Connect your ARTIK 530 or Raspberry Pi 3 board through USB to a Linux computer, and switch on SDB.
+2.  Right-click the project and select **Run As &gt; Tizen Native Project**.
+
+    ![Testing the sample](media/sample_test.png)
```

Comment:
Image is missing.

Reply:
done

Reply:
It looks this file is tangled from line 106. ` ``` ` is added so below lines are in wrong format.